### PR TITLE
people love a good commit

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: project/meetings.rst [hunk 1]
```diff
@@ -86,65 +86,6 @@ An overview of the whole week as well as a detailed schedule for each day of the
 |  17:30-18:00   |               |               |               |               |               |
 +----------------+---------------+---------------+---------------+---------------+---------------+
 
-
-Monday (8/22)
```
Comment (nabobalis) [inline project/meetings.rst:90]: This should be stored somewhere.

File: project/meetings.rst [hunk 2]
```diff
@@ -159,10 +100,6 @@ The meeting will be held in person at the `Dublin Institute for Advanced Studies
 
     <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2382.6655452906884!2d-6.247826284162118!3d53.33134147997581!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x48670ebc417667b9%3A0x2ebe337565685fca!2sDublin%20Institute%20for%20Advanced%20Studies!5e0!3m2!1sen!2sie!4v1651738610863!5m2!1sen!2sie" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
 
-.. raw:: html
-
-    <iframe src="https://www.google.com/maps/embed?pb=!4v1651748599429!6m8!1m7!1s8dG6i23a8Z54tk2qqX3uww!2m2!1d53.33157163944034!2d-6.246030024886875!3f141.65991100205062!4f-0.21975806055093017!5f0.7820865974627469" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
```
Comment (nabobalis) [inline project/meetings.rst:164]: I think it was time for the street view to go.

File: project/meetings.rst [hunk 3]
```diff
@@ -50,100 +50,64 @@ Sessions more relevant to a general audience will be held between 14:00 and 18:0
...
-
-Friday (8/26)
-+++++++++++++
-
-* Hack Day Ideas
-    - Element Workflow (Stuart)
-    - Website/Theme (Nabil)
-    - Refactor Map onto NDCube (Dan)
-    - Solar models package (Dan)
+.. raw:: html
```
Comment (nabobalis) [inline project/meetings.rst:53]: I might have gone mad.

**Comment by wtbarnes** [comment]
I now really wish I hadn't spent any amount of time constructing that table in RST 🤦‍♂️

**Comment by wtbarnes** [comment]
More seriously though, I'm not sure that the horizontal format is better?

**Comment by nabobalis** [comment]
I can invert it?

File: project/meetings.rst (hunk 3 above)
Comment (nabobalis) [reply project/meetings.rst:53]: There are some CSS issues with the theme, so I had to style things directly in the html as a work around.

**Comment by nabobalis** [comment]
> I can invert it?

I infact can.

**Pull request merged**